src/telemetry.ts

```diff
@@ -974,36 +974,46 @@ export class IEventNamePropertyMapping {
      */
     [Telemetry.DebugContinue]: TelemetryEventInfo<DurationMeasurement> = {
         owner: 'roblourens',
-        feature: ['Debugger']
-    } as any;
+        feature: ['Debugger'],
```

Comment:
If you don't need  the measure here `duration`, then you can change the call to this telemetry event from `capturePerfTelemetry` to captureUsageTelemetry`.
Personally I don't think the perf is useful, but I leave that upto you to decide.